Assignment2/PaulaPousa-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,68 @@
+{
+  "@context":{
+    "xsd": "http://www.w3.org/2001/XMLSchema#", 
+    "prop": "http://example.org#", 
+    "data": "http://example.org/data" 
+  }, 
+  
+  "@graph":[ 
+    {
+      "@id": "http:/example.org/Class01", 
+      "@type": "prop:Class01/", 
+      "prop:includes": [ 
+        { 
+          "@id": "http://example.org/Class01/Sensor029", 
+          "@type" : "Class01/Sensor029" 
+        }, 
+        { 
+          "@id": "http://example.org/Class01/Computer101", 
+          "@type" : "Class01/Computer101" 
+        } 
+      ]	
+    }, 
+    { 
+      "@id": "http://example.org/Class01/Sensor029/", 
+      "@type": "Class01/Sensor029", 
+      "prop:hasMeasurement": 
+        { 
+          "@id": "http://example.org/Class01/Sensor029/Measurement8401", 
+          "@type": "Sensor029:Measurement8401"	
+        } 
+    }, 
+    { 
+      "@id": "http://example.org/Class01/Computer101", 
+      "@type": "Class01:Computer101", 
+      "prop:hasOwner": 
+        { 
+          "@id": "http://example.org/Class01/Computer101/User10A", 
+          "@type": "Computer101:User10A" 
+        }      
+    }, 
+    { 
+      "@id": "http://example.org/Class01/Sensor029/Measurement8401", 
+      "@type": "Sensor029:Measurement8401", 
+      "prop:hasTemperature": 
+        { 
+          "@value": "29", 
+          "@type": "xsd:integer" 
+        } 
+      "prop:atTime": 
+        { 
+          "@value": "2010-06-12T12:00:12", 
+          "@type": "xsd:dateTime" 
+        } 
+    }, 
+    { 
+    "@id": "http://example.org/Class01/Computer101/User10A", 
+    "@type": "Computer101:User10A", 
+    "prop:hasName": 
+      { 
+        "@value": "Pedro", 
+        "@type": "xsd:string" 
+      }
+    } 
+  ] 
+}
+
+
+
```

Comment:
The structure of this file is ok. However, you have wrong "@type" definitions (except for the Class01 resource). You should use the prefix that has already defined ("prop") to provide correct type definitions. For example the resource Sensor029 might be "@type" : "prop:Sensor029" instead of "@type": "Class01/Sensor029" since "Class01" isn't a prefix.